eap/eap71-tx-recovery-s2i.json

```diff
@@ -155,9 +155,40 @@
              "name": "SPLIT_DATA",
              "value": "true",
              "required": false
+        },
+        {
+            "displayName": "Service account name",
+            "description": "Name of the service account used for this image",
+            "name": "SERVICE_ACCOUNT_NAME",
+            "value": "listing-pod",
+            "required": false
         }
     ],
     "objects": [
+        {
+            "apiVersion": "v1",
+            "kind": "ServiceAccount",
+            "metadata": {
+                "name": "${SERVICE_ACCOUNT_NAME}"
```

Comment:
I would recommend removing the parameter and just have it use something like `${APPLICATION_NAME}-account` or `${APPLICATION_NAME}-sa`.

Reply:
I see, thanks. Here you go.